src/main/flight/mixer.c

```diff
@@ -333,6 +333,28 @@ static servoMixer_t *customServoMixers;
 
 static motorMixer_t *customMixers;
 
+static uint16_t disarmEscOutput, minEscOutput, maxEscOutput, deadband3dHigh, deadband3dLow;
```

Comment:
`disarmEscOutput, minEscOutput, maxEscOutput` are misnomers, since, presumably, they also apply to brushed motors.

How about `disarmMotorOutput, minMotorOutput, maxMotorOutput` ?